packages/block-editor/src/hooks/content-lock-ui.js

```diff
@@ -156,14 +152,7 @@ export const withBlockControls = createHigherOrderComponent(
 						) }
 					</BlockSettingsMenuControls>
 				) }
-				<BlockEdit
-					{ ...props }
-					className={ classnames(
-						props.className,
-						isEditingAsBlocks &&
-							'is-content-locked-editing-as-blocks'
```

Comment:
This class isn't used in the editor. Probably a left over from the original PR.

Reply:
There is, however, a `.is-content-locked-temporarily-editing-as-blocks` class that has styles. Can you check whether it maybe should apply to this element? It could be a bug that it's not applied.

Reply:
Correct. The `.is-content-locked-temporarily-editing-as-blocks` use used to apply "Spotlight" styling to the block.

I double-checked this with the original PR (#43037), and the "edit mode" works as before.